### ISSUE TITLE
zero validation accuracy

### ISSUE THREAD

**Issue by hojjatb** [walsvid/VGG-TensorFlow#9]
during running the code, zero validation accuracy obtained for almost all  the steps. what mistake leads to this problem?
